shift_detector/checks/lda_check.py

```diff
@@ -54,28 +63,130 @@ def run(self, store) -> Report:
+        return plot_functions
+
+    @staticmethod
+    def paired_total_ratios_figure(column, df1, df2):
+        value_counts = pd.concat([df1['topics ' + column].value_counts(), df2['topics ' + column].value_counts()],
+                                 axis=1).sort_index()
+        value_ratios = value_counts.fillna(0).apply(axis='columns',
+                                                    func=lambda row: pd.Series([row.iloc[0] /
+                                                                                len(df1['topics ' + column]),
+                                                                                row.iloc[1] /
+                                                                                len(df2['topics ' + column])],
+                                                                               index=[str(column) + ' 1',
+                                                                                      str(column) + ' 2']))
+        axes = value_ratios.plot(kind='barh', fontsize='medium')
+        axes.invert_yaxis()  # to match order of legend
+        axes.set_title(str(column), fontsize='x-large')
+        axes.set_xlabel('ratio', fontsize='medium')
+        axes.set_ylabel('topics', fontsize='medium')
+        plt.show()
+
+    def word_cloud(self, column, topic_words, n_topics, lib):
+        custom_XKCD_COLORS = mcolors.XKCD_COLORS
+        # remove very light colors that are hard to see on white background
+        custom_XKCD_COLORS.pop('xkcd:yellowish tan', None)
+        custom_XKCD_COLORS.pop('xkcd:really light blue', None)
+        cols = [color for name, color in custom_XKCD_COLORS.items()]
+
+        cloud = WordCloud(background_color='white',
+                          width=2500,
+                          height=1800,
+                          max_words=10,
+                          collocations=False,
+                          color_func=lambda *args, **kwargs: cols[i],
+                          prefer_horizontal=1.0)
+        j = int(np.ceil(n_topics / 2))
+
+        fig, axes = plt.subplots(j, 2, figsize=(10, 10+n_topics))
+
+        for i, ax in enumerate(axes.flatten()):
+            if n_topics % 2 == 1 and i == n_topics:
+                fig.add_subplot(ax)
+                ax.axis('off')
+                break
+            fig.add_subplot(ax)
+            if lib == 'gensim':
+                topics = dict(topic_words[column][i][1])
+                cloud.generate_from_frequencies(topics, max_font_size=300)
+            else:
+                topics = ' '.join(set(topic_words[column][i]))
+                cloud.generate_from_text(topics)
+
+            ax.imshow(cloud)
+            ax.set_title('Topic ' + str(i+1), fontdict=dict(size=16))
+            ax.axis('off')
+
+        plt.axis('off')
+        plt.margins(x=0, y=0)
+        plt.show()
+
+    @staticmethod
+    def py_ldavis(column, lib, lda_models, dtm=None, vectorizer=None, corpus=None, dictionary=None):
```

Comment:
I read this as py_l_Davis and was wondering what it meant, until I realized it was supposed to be lda_vis.
I think it's better not to abbreviate